src/main/java/com/marklogic/appdeployer/command/ResourceManagerImpl.java

```diff
@@ -0,0 +1,161 @@
+package com.marklogic.appdeployer.command;
+
+import com.marklogic.client.ext.helper.LoggingObject;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Properties;
+
+/**
+ * Defines operations for managing whether a module needs to be installed or not.
+ */
+public class ResourceManagerImpl extends LoggingObject implements ResourceManager {
+
+	public static final String DEFAULT_FILE_PATH = "build/ml-gradle/resource-timestamps.properties";
+
+	private static MessageDigest md5Digest;
+	static {
+		try {
+			md5Digest = MessageDigest.getInstance("MD5");
+		} catch (NoSuchAlgorithmException e) {
+			e.printStackTrace();
```

Comment:
Use the LoggerFactory class to get a Logger to log this at the "warn" level with a message like "Unable to instantiate MD5 MessageDigest, will not be able to determine if resource files should not be deployed; cause:  " + e.getMessage().